pytorch_lightning/profiler/pytorch.py

```diff
@@ -131,41 +131,57 @@ def reset(self):
         self._prev_schedule_action: Optional[ProfilerAction] = None
         self._start_action_name: Optional[str] = None
 
+    @property
+    def is_training(self):
```

Comment:
These don't have to be public properties on the profiler since they're only used internally in the class